src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -173,8 +173,8 @@ private void testAddInstructorAction() throws Exception {
         
         courseEditPage.clickAddSectionLevelPrivilegesLink(newInstructorIndex);
         courseEditPage.clickSectionSelectionCheckBox(newInstructorIndex, 1, 1);
-        courseEditPage.clickViewStudentCheckBoxInSectionLevel(newInstructorIndex, 0);
-        courseEditPage.clickViewSessionResultsCheckBoxInSectionLevel(newInstructorIndex, 0);
+        courseEditPage.clickSectionLevelPrivilegeLink(newInstructorIndex, 0, 1);
+        courseEditPage.clickSectionLevelPrivilegeLink(newInstructorIndex, 0, 6);
```

Comment:
These numbers 1, 6, ... are all magic numbers. Perhaps in a separate issue, extract those to constants?